test/db/cmd/elf

```diff
@@ -16,3 +16,13 @@ EXPECT=<<EOF
 0x0804841d
 EOF
 RUN
+
+NAME=et_dyn no warnings for relocations
```

Comment:
do something here. like listing relocs with ir.

Reply:
nah, i just wanted to make sure `Warning: use -eio.cache=true to fix relocations` message was not shown. There is no need to do anything.

Reply:
ok